### ISSUE TITLE
Allow special case shorthand syntax

### ISSUE THREAD

**Issue by Qix-** [chalk/chalk-template#1]
I find myself using 256 colors quite a bit and having to do `ansi256(xxx)` in my template tags is less than ideal.

Since we don't support any numeric colors as methods, would it make sense to allow the shorthands for RGB + 256 such that the following works?

```javascript
chalk`{#FF0000 foo}` // truecolor output
chalk`{127 foo}` // ansi256 output
```

**Comment by sindresorhus** [comment]
👍

**Comment by Qix-** [comment]
Landed in `0.3.0`. Thanks @Qix-!

**Comment by Qix-** [comment]
You're welcome @Qix- <3
